openff/recharge/cli/reconstruct.py

```diff
@@ -122,5 +114,22 @@ def reconstruct(
             "qcelemental": qcelemental.__version__,
         },
     )
+    
+    with ProcessPoolExecutor(max_workers=n_processors, mp_context=get_context("spawn")) as pool:
+ 
+        futures = [
+            pool.submit(
+                functools.partial(_process_result, grid_settings=grid_settings),
```

Comment:
This looks great! I don't think we need to use `functools.partial` anymore though we should just be able to pass the function, args and kwars to `pool.submit`.